File: Backend/src/main/java/com/feedle/feedleapi/Networking/AddConversationRequest.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.Conversation;

public class AddConversationRequest extends Request {
    private Conversation conversation;
    private int creatorId;
    private int withWhomId;

    public AddConversationRequest(Conversation conversation, int creatorId, int withWhomId) {
        super(RequestType.AddConversation);
        this.creatorId = creatorId;
        this.conversation = conversation;
        this.withWhomId = withWhomId;
    }

    public Conversation getConversation() {
        return conversation;
    }

    public int getCreatorId() {
        return creatorId;
    }

    public int getWithWhomId() {
        return withWhomId;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/AddConversationRequest.cs
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class AddConversationRequest : Request
    {
        [JsonPropertyName("conversation")]
        public Conversation Conversation { get; set; }
        [JsonPropertyName("creatorId")]
        public int CreatorId { get; set; }
        [JsonPropertyName("withWhomId")]
        public int WithWhomId { get; set; }

        public AddConversationRequest(Conversation conversation, int creatorId, int withWhomId) : base(RequestType.AddConversation)
        {
            Conversation = conversation;
            CreatorId = creatorId;
            WithWhomId = withWhomId;
        }
        
    }
}